doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -1350,10 +1350,10 @@ Generation of inversion principles with ``Derive`` ``Inversion``
    Generates an inversion lemma for the
    :tacn:`inversion` tactic.  :token:`ident` is the name
    of the generated lemma.  :token:`one_term` should be in the form
-   :token:`qualid` or :n:`(forall {+ @binder }, @qualid @term)` where
+   :token:`qualid` or :n:`(forall {+ @binder }, @qualid {+ @term })` where
```

Comment:
I don't know this command.  Does the `@qualid @term` correspond to a function application?  If that's the intention, then it should be this so that non-trivial arguments have to be parenthesized.  And maybe better to be explicit that the number of terms should match the number required for `@qualid`?

```suggestion
   :token:`qualid` or :n:`(forall {+ @binder }, @qualid {+ @one_term })` where
```

It would be nice to have a short example here (in part, to convince me that what you say is correct rather than take it on faith).